File: despatch.Domain/Models/Default/Request/ContactRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class ContactRequest : BaseEntityRequest
    {
        public string UserId { get; set; }
        public string FriendId { get; set; }
        public string ContactDataId { get; set; }


        public virtual UserRequest User { get; set; }
        public virtual UserRequest Friend { get; set; }
        public ContactDataRequest ContactData { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataRequest } from './contact-data-request';
import { UserRequest } from './user-request';
export interface ContactRequest extends BaseEntityRequest{
  contactData?: ContactDataRequest;
  contactDataId?: string;
  friend?: UserRequest;
  friendId?: string;
  user?: UserRequest;
  userId?: string;
}
